### ISSUE TITLE
[BUG] RFT issue with `import xxx.xxx` imports

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1087]
When importing a file using the following format: `import xxx.xxx`, Pyarmor renames the import incorrectly.

**Executed command:** `pyarmor -d gen --enable-rft .\pyarmor_test\`

**pyarmor_test\main.py**
```python
import pyarmor_test.class_test


def testing():
    pyarmor_test.class_test.MainClass().test()
    pyarmor_test.class_test.SubClass().test()


if __name__ == "__main__":
    testing()
```

**pyarmor_test\class_test.py**
```python
class MainClass:
    def __init__(self):
        pass

    def test(self):
        print("MainClass.test")


class SubClass(MainClass):
    def __init__(self):
        super().__init__()

    def test(self):
        print("SubClass.test")

```

**Runtime error:**
```
Traceback (most recent call last):
  File "C:\Users\alec\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\alec\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "<frozen __main__>", line 3, in <module>
  File "<frozen pyarmor_test.main>", line 10, in <module>
  File "<frozen pyarmor_test.main>", line 5, in pyarmor__6
AttributeError: module 'pyarmor_test.class_test' has no attribute 'class_test'
```

**Comment by jondy** [comment]
Got it, thanks.

**Comment by jondy** [comment]
Fixed it in `8.1.dev5`
